### DISCUSSION TITLE
Control border inconsistencies

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/microsoft-ui-xaml#7449] [General]
A lot of controls got updated to have borders, but they're pretty inconsistent, especially in dark mode.
Some of them have light borders, and the others have dark ones. What is the intended design?

Light borders in dark mode do provide better contrast, especially for windows and flyouts, but some borders are meant to look like very small shadows.
There are sometimes controls of the same kind that are inconsistent, such as Flyout and TeachingTip.

There are also some buttons that don't have a border when selected. This is the case with NavigationView items, while Windows 11 taskbar buttons do have borders.

**Comment by StephenLPeters** [comment]
@gabrielconl can you provide screenshots of the issues you are calling attention to?

**Comment by ghost** [comment]
TeachingTip vs Flyout:
![image](https://user-images.githubusercontent.com/84711285/133250241-983d46b3-30d3-45ea-b9f8-909d4e35be12.png)
![image](https://user-images.githubusercontent.com/84711285/133250253-e850245e-ab6e-499c-b5ba-de38bb4ec9e4.png)

Button vs SplitButton vs InfoBar
![image](https://user-images.githubusercontent.com/84711285/133250453-513864b3-9195-434d-8cc1-c9dfedbec596.png)
![image](https://user-images.githubusercontent.com/84711285/133250481-0df75d4f-93e2-4eea-a110-99bf4627e867.png)
![image](https://user-images.githubusercontent.com/84711285/133251007-40a99a92-8c39-427f-898a-d8cbff2dfbf8.png)

CommandBarFlyout vs MenuFlyout
![image](https://user-images.githubusercontent.com/84711285/136381293-cb7889e8-f1ab-4a65-b600-100794447ddd.png)

**Comment by StephenLPeters** [comment]
@alejo4000 what do you think?

**Comment by ghost** [comment]
A couple more things:
Menu items that are normally transparent don't have borders when selected/hovered, but on the Windows 11 taskbar they do:
![image](https://user-images.githubusercontent.com/84711285/136670947-a73420d7-32b6-44b4-8026-bfb96a4ea5d9.png)
![image](https://user-images.githubusercontent.com/84711285/136670917-c6f8f7a4-61a3-4901-82b4-00cd136df9d1.png)

AppBarToggleButton has a border when selected, but DatePicker doesn't have one:
![image](https://user-images.githubusercontent.com/84711285/136670996-86bfe122-903d-418c-8aee-468855505428.png)
![image](https://user-images.githubusercontent.com/84711285/136671025-ca767627-865e-4ada-9111-1aa2bc120564.png)

I think it would make sense for items to have a border, at least when selected, since they're supposed to be elevated.

**Comment by StephenLPeters** [comment]
@alejo4000 fyi
